packages/callout/src/Callout.tsx

```diff
@@ -21,33 +19,33 @@ export const Variant = {
 export type Variant = typeof Variant[keyof typeof Variant];
 
 const baseStyle = css`
-  background-color: ${uiColors.white};
-  border-radius: 6px;
-  box-shadow: 0px 2px 5px 0 ${transparentize(0.9, uiColors.gray.dark3)};
-  overflow: hidden;
+  font-family: Euclid Circular A, ‘Helvetica Neue’, Helvetica, Arial, sans-serif; // TODO: Refresh – remove when fonts are updated
+  background-color: ${palette.white};
+  border-radius: 12px;
   position: relative;
 
   &:after {
     content: '';
     position: absolute;
-    width: 4px;
-    left: 0px;
+    width: 13px;
+    left: -1px;
     top: -1px;
     bottom: -1px;
-    border-radius: 6px 0px 0px 6px;
+    border-radius: 12px 0px 0px 12px;
   }
 `;
 
 const headerStyle = css`
-  font-family: Akzidenz, ‘Helvetica Neue’, Helvetica, Arial, sans-serif;
   font-size: 12px;
-  font-weight: bold;
-  letter-spacing: 0.8px;
+  font-weight: 700;
```

Comment:
I think the mocks are `Medium` (or `500`)

Reply:
If I set to `500` it begins to look thinner than the design. The top is `500` and the bottom is `700`:
<img width="119" alt="Screen Shot 2022-01-24 at 11 59 59 AM" src="https://user-images.githubusercontent.com/5740159/150828952-c2bfc5dc-6392-4846-a6c9-e605b1f2c766.png">

<img width="120" alt="Screen Shot 2022-01-24 at 11 59 37 AM" src="https://user-images.githubusercontent.com/5740159/150828875-1bf5bfad-1ab2-4628-874c-51e2ed6390ca.png">

This is from Figma:
<img width="120" alt="Screen Shot 2022-01-24 at 12 02 37 PM" src="https://user-images.githubusercontent.com/5740159/150829404-c1911892-5372-4a2e-a0cb-38af7f60c135.png">

Reply:
Try updating from `main`. I think the font definitions have changed slightly

Reply:
@TheSonOfThomp I do have the latest from `main`, maybe we can leave this for visual QA and see what the designers think?

Reply:
Figma doesn't use the browser's built in rendering, so some fonts might appear a bit lighter or heavier in one or the other. We should probably resolve this in QA and follow the Figma specifications (even if the rendering is a bit off).